app/lib/mini-course.cjsx

```diff
@@ -107,6 +109,8 @@ module.exports = React.createClass
     optOut: false
 
   componentDidMount: ->
+    @props.geordi.makeHandler 'tutorial-completion'
```

Comment:
From memory, `geordi.makeHandler` returns a function so you probably do need to assign it to an event listener somewhere, in order for it to be called.